src/bin/loadtest.rs

```diff
@@ -70,12 +70,26 @@ fn main() {
                 for idx in 0..jobs {
                     if idx % 2 == 0 {
                         // push
-                        let mut job = Job::new(
-                            "SomeJob",
-                            vec![serde_json::Value::from(1), "string".into(), 3.into()],
-                        );
-                        job.priority = Some(rng.gen_range(1..10));
-                        job.queue = QUEUES.choose(&mut rng).unwrap().to_string();
+                        let job = match idx % 4 {
+                            // Solely to demo how 'JobBuilder' works in userland
```

Comment:
This feels like it belongs in a code block in the documentation. The `loadtest` binary isn't really intended as user-facing documentation, but rather as a stand-lone tool.

Reply:
restored